Source/LibationFileManager/AudibleFileStorage.cs

```diff
@@ -126,7 +126,16 @@ protected override List<LongPath> GetFilePathsCustom(string productId)
                     BookDirectoryFiles = new BackgroundFileSystem(BooksDirectory, "*.*", SearchOption.AllDirectories);
 
             var regex = GetBookSearchRegex(productId);
-            return BookDirectoryFiles.FindFiles(regex);
+
+            //Find all extant files matching the priductID
```

Comment:
Ah, the ever expanding complexity to save users from themselves. (btw -- typo: priductID)

Reply:
Yup! I Always keep the asin in my naming templates, but I had removed it when helping another user and discovered that I could not convert liberated m4bs to mp3 because it couldn't find them.